src/service_args.py

```diff
@@ -14,7 +15,7 @@ def __init__(self, config: ConfigData, relay_rpc_urls: dict):
         self._relay_rpc_urls = relay_rpc_urls
         self._chain_spec_url = config.get('chain-spec-url')
         self._local_relaychain_spec_url = config.get('local-relaychain-spec-url')
-        self.__check_service_args(service_args)
+        self._runtime_wasm_override = True if config.get('wasm-runtime-url') else False
         self.service_args_list = self.__service_args_to_list(service_args)
         self.__check_service_args(self.service_args_list)
```

Comment:
Before, `__check_service_args()` was run twice, we're not losing any functionality by removing the first call to it?